<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1232</width>
    <height>476</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DiodeFinder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="image2">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/new/prefix/img1.jpg</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="image1">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/new/prefix/img1.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="image3">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/new/prefix/img1.jpg</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="startBtn">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="visualizationBtn">
        <property name="text">
         <string>3D</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Blur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderBlur">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Median</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderMedian">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Picking difference</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderDiff">
        <property name="value">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Triangle ratio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderRatio">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Epsilon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderEpsilon">
        <property name="value">
         <number>8</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>H+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderHU">
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>L+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderLU">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>150</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>S+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderSU">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>H -</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderHD">
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="value">
         <number>-20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>L -</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderLD">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>S -</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderSD">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>230</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lblParams">
      <property name="text">
       <string>Params:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblResults">
      <property name="text">
       <string>Results:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderHU</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderLU</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderSU</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderSD</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderLD</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderHD</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderBlur</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMedian</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>mouseClickEvent()</signal>
   <receiver>MainWindow</receiver>
   <slot>onClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderDiff</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderHD</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderHU</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderLD</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderLU</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderSD</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1033</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderSU</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1033</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderBlur</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderDiff</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1073</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderMedian</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderEpsilon</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1149</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderEpsilon</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1149</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderRatio</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>942</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderRatio</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>942</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>visualizationBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onVisualizationStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>mouseClickEvent()</signal>
  <slot>onStart()</slot>
  <slot>onSliderChange(int)</slot>
  <slot>onClick()</slot>
  <slot>onSliderReleased()</slot>
  <slot>onVisualizationStart()</slot>
 </slots>
</ui>
